client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SecurityDocumentationIT.java

```diff
@@ -131,7 +131,7 @@ public void testPutUser() throws Exception {
         }
         {
             byte[] salt = new byte[32];
-            SecureRandom.getInstanceStrong().nextBytes(salt);
+            random().nextBytes(salt);
```

Comment:
I didn't think about this on first look, but can you add a comment that this test intentionally uses a reproducible random and not a securerandom

Reply:
I pushed 16eceac5403a37e3df0ced0ff84ec4afda61966c.